contracts/lib/IdentityBase.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.16;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "../interfaces/IState.sol";
+import "../lib/ClaimBuilder.sol";
+import "../lib/OnChainIdentity.sol";
+
+// /**
+//  * @dev Contract managing onchain identity
+//  */
+contract IdentityBase is OwnableUpgradeable {
```

Comment:
Base contract should not be `OwnableUpgradable`. It should be without parents, so that other contracts could inherit it without becoming OwnableUpgradable themselves.